### ISSUE TITLE
universal IR

### ISSUE THREAD

**Issue by ddddddddggfekzurkj** [flipperdevices/flipperzero-firmware#4007]
### Description of the feature you're suggesting.

add to the universal IR remote control the ability to pause, scroll to the previous or next signal, save it and send the desired signal, like in the sub-ghz bruteforcer application

### Anything else?

добавте в универсальный ик пульт возможность приостановить, перелистнуть на прошлый или следующий сигнал, сохранить его и  отправить нужный сигнал типо как в приложении sub-ghz bruteforcer

**Comment by skotopes** [comment]
@portasynthinca3 wanna try something new?

**Comment by portasynthinca3** [comment]
Hey @ddddddddggfekzurkj! Please install the firmware from #4085 ("Web/App Updater" option) and check whether it's what you wanted.

Пожалуйста, установите прошивку из #4085 (пункт "Web/App Updater") и сообщите, это ли Вы просили.

**Comment by ddddddddggfekzurkj** [comment]
yes @portasynthinca3 this is what I asked for, it turned out really cool, thank you

да @portasynthinca3 это то что я просил получилось очень круто спасибо

**Comment by agathakazar** [comment]
This is great! 
Would it be possible to have the protocol and the hexes be displayed in this pause menu? 
Much more convenient than going to ctrl+f the tv.ir file

**Comment by agathakazar** [comment]
Also, Flipper hangs when I `ir universal tv Power` from the cli, feels like it's related (does it warrant a new issue? no idea)
